resources/client_starter.py

```diff
@@ -69,20 +69,27 @@ def make_condor_job_description(  # pylint: disable=too-many-arguments
     #   entrypoint, and loading the icetray env file
     #   directly from cvmfs messes up the paths" -DS
 
-    environment = ""
+    vars = []
+
     if not os.getenv("RABBITMQ_HEARTBEAT"):
-        environment = f"RABBITMQ_HEARTBEAT=600 {environment}"
+        vars.append("RABBITMQ_HEARTBEAT=600")
     if not os.getenv("EWMS_PILOT_QUARANTINE_TIME"):
-        environment = f"EWMS_PILOT_QUARANTINE_TIME=1800 {environment}"
+        vars.append("EWMS_PILOT_QUARANTINE_TIME=1800")
+
+    for prefix in ["APPTAINERENV_", "SINGULARITYENV_"]:
+        vars.append(f"{prefix}I3_DATA=/cvmfs/icecube.opensciencegrid.org/data")
+
+    environment = " ".join(vars)
```

Comment:
You could use this `join()` directly in the actual dictionary assignment below. But it's not a deal breaker

Reply:
True, but conceptually I like the separation. If in the future we decide to build the "environment" string differently we do not have to change the code in the submit_dict block.